src/parser/jobs/ast/modules/ArcanaTracking/ArcanaTracking.tsx

```diff
@@ -338,7 +338,7 @@ export default class ArcanaTracking extends Analyser {
 				...event,
 			},
 			drawState: undefined,
-			sealState: CLEAN_SEAL_STATE,
+			sealState: lastCardState.sealState,
```

Comment:
Doesn't dying remove all your seals? This looks like it would carry them over post-death... or did I miss something (._.)

Reply:
It now carries them over post death (but not when changing instances)
The only reason I remembered this is because I saw a log where someone died and then immediately used astrodyne which wouldn't have been possible without retaining seals

![image](https://user-images.githubusercontent.com/89435408/147884360-36a89f4b-0d33-4025-b6f1-02ed4dd96db0.png)

Reply:
Found an irl example to show that this was implemented this patch if you're curious
https://www.fflogs.com/reports/jtJwWN4aH9XRMYxG

https://xivanalysis.com/fflogs/jtJwWN4aH9XRMYxG/20/1104
Click the last zodiark

![image](https://user-images.githubusercontent.com/89435408/147884751-a6db409a-5e94-45b0-b371-2169fb2133f1.png)

Reply:
Dear YoshiP, that's bonkers
- sushi